core/store/migrate/migrations/0226_log_poller_index_drop.sql

```diff
@@ -0,0 +1,5 @@
+-- +goose Up
+drop index evm.evm_logs_idx_created_at;
```

Comment:
This index shouldn't be used anywhere, because there are no more queries that rely on that field. I've also checked index utilization for CCIP to double check

```sql
 evm.evm_logs_by_timestamp                        |        21272200 |  3524320643 |     3522526611
 evm.logs_pkey                                    |        12636814 |    11596801 |       11596715
 evm.idx_evm_logs_ordered_by_block_and_created_at |         4240547 | 20127987563 |    15762720980
 evm.evm_logs_idx                                 |         7208398 |    24196816 |       22603336
 evm.evm_logs_idx_data_word_one                   |               0 |           0 |              0
 evm.evm_logs_idx_data_word_three                 |           27315 |   390229380 |              0
 evm.evm_logs_idx_data_word_two                   |               0 |           0 |              0
 evm.evm_logs_idx_topic_four                      |               0 |           0 |              0
 evm.evm_logs_idx_topic_three                     |               0 |           0 |              0
 evm.evm_logs_idx_topic_two                       |          311176 |   104034706 |          17779
 evm.evm_logs_idx_created_at                      |               0 |           0 |              0
 evm.evm_logs_idx_tx_hash                         |             745 |         932 |            922
 evm.evm_logs_idx_data_word_four                  |               0 |           0 |              0


```

Reply:
SonarQube is complaining about the lack of capitalization, will be cleaner to look at for the future if we address it:
```go
--- drop index evm.evm_logs_idx_created_at;
+++ DROP INDEX evm.evm_logs_idx_created_at;
```